addons/l10n_ch/tests/test_partner_bank_isr.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Camptocamp SA
```

Comment:
If I may give an opinion @oco-odoo I agree with fact test should be about flow and corner cases. Here in Switzerland we experienced many flow issue because eof this constraint being wrong.

No payment can be made!

This is a very big issu for our customers here of course. So Please, could you reconsider including the tests?